hathor/conf/get_settings.py

```diff
@@ -41,18 +39,22 @@ def HathorSettings() -> Settings:
     """
     Returns the configuration named tuple.
 
-    Tries to get the configuration from a python module in the 'HATHOR_CONFIG_FILE' env var, which will be deprecated.
     If not found, tries to get it from a yaml filepath in the 'HATHOR_YAML_CONFIG', which will be the new standard.
```

Comment:
```suggestion
    Gets the configuration from a yaml filepath in the 'HATHOR_YAML_CONFIG'.
    Before it, tries to get from a python module, which was deprecated, and throws a RuntimeError in case it's found.
```